metron-docker/install-metron.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+METRON_DOCKER_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $METRON_DOCKER_ROOT/.env
+METRON_PLATFORM_ROOT=$METRON_DOCKER_ROOT/../metron-platform
+if [ $# -gt 0 ] && [ $1 == "-b" ]
+    then cd $METRON_PLATFORM_ROOT && mvn clean package -DskipTests
+fi
+mkdir -p $METRON_DOCKER_ROOT/mysql/enrichment/
```

Comment:
Are these directories in a gitignore somewhere? Seems like we'll end up with new untracked files when we run the install command.